src/components/Extension/DocuSignExtension/actions/EmbeddedSigningAction/FinishedMessage.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from 'react'
+import { useTranslation } from 'next-i18next'
+import { Button } from '@awell_health/ui-library'
+import { DocuSignEvent } from '../../types'
+import { LoadingPage } from '../../../../LoadingPage'
+
+interface FinishedMessage {
+  event: DocuSignEvent
+  finishSigning: () => void
+}
+
+export const FinishedMessage: FC<FinishedMessage> = ({
+  event,
+  finishSigning,
+}) => {
+  const { t } = useTranslation()
+
+  switch (event) {
+    case DocuSignEvent.SIGNING_COMPLETE:
+      return (
+        <LoadingPage title={t('activities.docu_sign.finished_sign_document')} />
+      )
+    case DocuSignEvent.TTL_EXPIRED:
```

Comment:
How long is a signing URL valid? We need to add this is a big fat disclaimer in the README of the extension.

---

EDIT: I see it's in there - https://github.com/awell-health/awell-extensions/tree/main/extensions/docuSign